src/lib/isMobilePhone.js

```diff
@@ -50,7 +50,6 @@ const phones = {
   'en-ZM': /^(\+?26)?09[567]\d{7}$/,
   'en-ZW': /^(\+263)[0-9]{9}$/,
   'es-AR': /^\+?549(11|[2368]\d)\d{8}$/,
-  'es-BO': /^(\+?591)?(6|7)\d{7}$/,
```

Comment:
Looks you like you are overwriting some previous work, please rebase correctly.

Reply:
fixed

Reply:
Still... I'm I seeing wrongly? There should be just about 3 files in the diff I think.